initrd/bin/config-gui.sh

```diff
@@ -55,6 +55,7 @@ while true; do
   whiptail --clear --title "Config Management Menu" \
     --menu "This menu lets you change settings for the current BIOS session.\n\nAll changes will revert after a reboot,\n\nunless you also save them to the running BIOS." 20 90 10 \
     'b' ' Change the /boot device' \
+    'u' ' Change the USB boot device' \
```

Comment:
This was removed in the USB autodetection PR #548 you just merged.

Reply:
just saw it at the same time. forced pushed